src/main/resources/avro/variants.avdl

```diff
@@ -180,8 +264,10 @@ record Variant {
   /**
   The reference on which this variant occurs.
   (e.g. `chr20` or `X`)
+
+  If the API server supports the "classic" mode, this field must not be null.
   */
-  string referenceName;
```

Comment:
It would have to be a `Segment` to hold the end, and then if each `Variant` has a `Segment`, why not just use a `Path` and support the whole graph model?

I kept the original fields around for backwards compatibility. Anything that was a valid `Variant` before this PR should be a valid `Variant` afterwards.